manifests/repos.pp

```diff
@@ -8,11 +8,14 @@
 #
 # @param architecture
 #
+# @param keyring
```

Comment:
Should add documentation, not just list it here.

Reply:
```suggestion
# @param keyring
#   Absolute path to a file containing the PGP keyring used to sign this repository. Value is used to set signed-by on the source entry.
#   See https://wiki.debian.org/DebianRepository/UseThirdParty for details.
```